modules/acs-secured-cluster-services-overview.adoc

```diff
@@ -17,3 +17,5 @@ In addition, Sensor is responsible for all cluster interactions, such as applyin
 It collects container runtime and network activity information and sends the collected data to Sensor.
 * *Scanner*: In Kubernetes, the secured cluster services include Scanner-slim as an optional component.
 However, on {ocp}, {product-title-managed-short} installs a Scanner-slim version on each secured cluster to scan images in the {ocp} integrated registry and optionally other registries.
+* *Scanner V4 DB*: This component is installed if Scanner V4 is enabled. This database stores information for Scanner V4, including index reports. For best performance, configure a persistent volume claim (PVC) for Scanner V4 DB.
```

Comment:
Scanner V4 Indexer seems to be missing now. Can you re-add it? Only Indexer, not Matcher